docs/getting_started.rst

```diff
@@ -46,6 +46,19 @@ If you have installation problems, consider opening an
 `issue <https://github.com/rl-institut/oemof-B3/issues>`_.
 
 
+How to install geopandas under Windows
+--------------------------------------
+The installation of geopandas on Windows can be challenging. According to the geopandas documentation (https://geopandas.org/getting_started/install.html) there are multiple ways to install it. We recommend to use the conda-forge channel:
```

Comment:
Can you begin with a sentence explaining for what geopandas is needed?

Reply:
Currently, it is only used for preparation of conventional plant data. There could be other uses in the future. Thus: "Geopandas is necessary for a small subset of the modeling steps. Therefore it is not part of the default requirements." Or similar

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/89/commits/1480f7d609c01e07c3bc9a239d73d21f39c3854b.